include/ofi_atom.h

```diff
@@ -123,6 +124,35 @@ typedef atomic_long	ofi_atomic_int64_t;
 		ATOMIC_IS_INITIALIZED(atomic);								\
 		return (int##radix##_t)atomic_fetch_sub_explicit(&atomic->val, val,			\
 								 memory_order_acq_rel) - val;		\
+	}												\
+	/**												\
+	 *  Compare and swap, strong version - should not be used by loops				\
```

Comment:
I would remove the loop comment.  It's fine to use the strong version in loops, and may be needed unless the loop ignores spurious failures.